src/main/java/uk/gov/pay/webhooks/validations/DomainNotOnAllowListException.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.pay.webhooks.validations;
+
+public class DomainNotOnAllowListException extends RuntimeException {
```

Comment:
Maybe this should be renamed `CallbackUrlDomainNotOnAllowListException` to match the other exceptions.